cms/static/cms/js/modules/cms.sideframe.js

```diff
@@ -251,6 +257,15 @@ var CMS = window.CMS || {};
                     that._addToHistory(this.contentWindow.location.href);
                 });
 
+                // prevent scrolling in underlying page
+                iframe.on('mouseenter mouseleave', function (e) {
```

Comment:
I'm not sure if we should use this on ``_open`` and ``_close`` might be risky, what do you think @vxsx ?